examples/ignition/bootkube-worker.yaml

```diff
@@ -41,7 +41,8 @@ systemd:
           --volume opt-cni-bin,kind=host,source=/opt/cni/bin \
           --mount volume=opt-cni-bin,target=/opt/cni/bin \
           --volume var-log,kind=host,source=/var/log \
-          --mount volume=var-log,target=/var/log"
+          --mount volume=var-log,target=/var/log \
+          --hosts-entry=host"
```

Comment:
This will start failing as soon as the wrapping script introduces the same flag: https://github.com/rkt/rkt/blob/053f014c38bf0408f68533837bd457034a907afb/rkt/run.go#L650

/cc @squeed

Reply:
Hm, but until that lands some clusters need this to have functional `port-forward` :(